packages/models-library/src/models_library/api_schemas_webserver/projects_ui.py

```diff
@@ -87,6 +87,7 @@ class StudyUI(BaseModel):
     slideshow: dict[NodeIDStr, SlideshowUI] | None = None
     current_node_id: Annotated[NodeID | None, Field(alias="currentNodeId")] = None
     annotations: dict[NodeIDStr, AnnotationUI] | None = None
+    template_type: Annotated[str | None, Field(alias="templateType")] = None
```

Comment:
TIP:  

1. if you change the inheritance, all alias are automatic 
2. instead of a `str` you can use a `Literal` to enforce a group of names for the template

```python

from ._base import OutputSchema

class StudyUI(OutputSchema): # <-----
    # Model fully controlled by the UI and stored under `projects.ui`
    icon: HttpUrl | None = None

    workbench: dict[NodeIDStr, WorkbenchUI] | None = None
    slideshow: dict[NodeIDStr, SlideshowUI] | None = None
    current_node_id: NodeID | None = None
    annotations: dict[NodeIDStr, AnnotationUI] | None = None
    template_type: Literal["hypertool", "pedro", "odei"] | None = None # <-----
```